R/lockfile-external.R

```diff
@@ -0,0 +1,99 @@
+renv_lockfile_external <- function() {
+  list(
+    quarto = quarto_version(),
+    pandoc = pandoc_version()
+  )
+}
+
+
+# pandoc ------------------------------------------------------------------
+
+pandoc_path <- function() {
```

Comment:
```suggestion
renv_pandoc_path <- function() {
```

By convention, (almost) all internal functions in should have an `renv_` prefix, unless they're generally-applicable utility functions. Can we continue that convention?